### ISSUE TITLE
Bug - Mobile menu chevron rotates around wrong point

### ISSUE THREAD

**Issue by maxious** [AusDTO/gov-au-ui-kit#232]
## What I did

Implemented v1.6.0 local nav markup:

```
<aside class="sidebar">
<button aria-controls="local-nav" class="local-nav-toggle">Menu</button><nav aria-label="main navigation" class="local-nav" id="local-nav" aria-expanded="false">
<h1 class="is-visuallyhidden">
Menu
</h1>
<ul>
<li>
<a class="is-active" href="/comms-and-arts">
Comms and Arts
</a>
<ul>
<li>
<a href="/comms-and-arts/about-us">
About Us
</a>
<ul>

</ul>
</li>

</ul>
</li>
</ul>
</nav>
</aside>
```
### What browser and device I was using

Macbook 13", Chrome Version 52.0.2743.82 (64-bit)
## What I expected to see

Nice rotation like gov.au/alpha
## What I saw

![50cf2870feb8fcb0f6b88e7fbf8dd1e7](https://cloud.githubusercontent.com/assets/81432/17280518/e5772bd6-57d4-11e6-8c8f-6bc073b01353.gif)

**Comment by TrebBrennan** [comment]
Thanks Alex.
**Note** button styles are incorrect here too.

**Comment by hannah-ustwo** [comment]
Funny little chevron...

Created an issue in JIRA, put directly into our current sprint. https://govausites.atlassian.net/browse/DESIGN-281

**Comment by hannah-ustwo** [comment]
Addressing on #236

**Comment by klepas** [comment]
This should fix https://github.com/AusDTO/gov-au-ui-kit/issues/232

**Comment by klepas** [comment]
Fixed in 1.7.0.
